optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/tri/TriConstraintStream.java

```diff
@@ -196,6 +198,80 @@
         return join(otherClass, AbstractQuadJoiner.merge(joiners));
     }
 
+    // ************************************************************************
+    // Group by
+    // ************************************************************************
+
+    /**
+     * Runs all tuples of the stream through a given @{@link TriConstraintCollector} and converts them into a new
```

Comment:
the other javadocs start with "convert tri into uni".
For maintenance, I recommend we keep the javadocs as much copy pasted as possible (same structure), so "convert into what" before "how" (like in other javadocs, this javadocs does it in reverse order).